src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/SkillsetsOperations.cs

```diff
@@ -185,7 +185,7 @@ public async Task<AzureOperationResponse<Skillset>> CreateOrUpdateWithHttpMessag
 
             if (customHeaders != null)
             {
-                foreach(var _header in customHeaders)
+                foreach (var _header in customHeaders)
```

Comment:
I think this is some VS weirdness as part of my commit `3bcaaf55`, where I replaced NERSkill with EntitySkill... that commit messed up the whitespacing in this file...

I'll regenerate this from swagger and add a different commit. Thanks for the spot!

Reply:
Nobody should ever be editing the generated code by hand. Was this the result of a global find-and-replace or something?

Reply:
Nope. AFAIK I must've just opened the file in VS to view it and then must've hit CTRL-S and hit quit... Some plugin that i have that does this auto whitespace formatting would've recked havoc...

What is expected however is the "re-ordering" of the methods, because I changed the order in swagger to be consistent with other operations